docs/modules/ROOT/pages/pallets/pallet_xcm.adoc

```diff
@@ -0,0 +1,340 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= pallet_xcm
+
+Branch/Release: `release-polkadot-v{1.3.0}`
+
+== Purpose
+
+`pallet-xcm` is responsible for filtering, routing, and executing incoming XCM.
+
+== Glossary
+
+** `Barrier` -- whether to execute the XCM call at all
```

Comment:
this explanation is a bit confusing imo. The reason for me is: `whether to execute the XCM call at all` is a boolean, yet `barrier` is probably not a boolean. 

If `barrier` is what I think, maybe rewording this as `set of criteria on whether to execute the XCM calls` would be better.

I may have misinterpreted the whole thing, feel free to correct me